data/json/items/gunmod/mechanism.json

```
@@ -70,6 +70,27 @@
     "mode_modifier": [ [ "DEFAULT", "auto", 6 ] ],
     "flags": [ "INSTALL_DIFFICULT" ]
   },
+  {
+    "id": "gswitch",
+    "type": "GUNMOD",
+    "name": { "str": "glock switch" },
+    "description": "Patented in 1998 the Glock auto-sear or switch as it is called was illegal due to being able to turn Glock firearms full auto by applying force to a Glock's trigger bar. The switch allows for selective fire modes, a single shot or when flipped, a six round full auto fire.",
```

```suggestion
    "description": "Patented in 1998 the Glock auto-sear or switch as it is called was illegal due to being able to turn Glock firearms full auto by applying force to a Glock's trigger bar.  The switch allows for selective fire modes, a single shot or when flipped, a six round full auto fire.",
```
double space after full stops.